editor/plugins/abstract_polygon_2d_editor.cpp

```diff
@@ -261,7 +261,7 @@ bool AbstractPolygon2DEditor::forward_gui_input(const Ref<InputEvent> &p_event)
 		Transform2D xform = canvas_item_editor->get_canvas_transform() * _get_node()->get_global_transform();
 
 		Vector2 gpoint = mb->get_position();
-		Vector2 cpoint = _get_node()->get_global_transform().affine_inverse().xform(canvas_item_editor->snap_point(canvas_item_editor->get_canvas_transform().affine_inverse().xform(mb->get_position())));
+		Vector2 cpoint = _get_node()->to_local(canvas_item_editor->snap_point(canvas_item_editor->get_canvas_transform().affine_inverse().xform(mb->get_position())));
```

Comment:
Maybe I'm wrong, but shouldn't second `get_canvas_transform().affine_inverse().xform()` be replaced to `to_local()` too?

I mean in `canvas_item_editor->get_canvas_transform().affine_inverse().xform(mb->get_position())`